core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -133,7 +133,8 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
   this.serverConfig.setProperty(KafkaConfig.AuthorizerClassNameProp, classOf[SimpleAclAuthorizer].getName)
   // Some needed configuration for brokers, producers, and consumers
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicPartitionsProp, "1")
-  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "1")
+  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
```

Comment:
Just to make sure I understand it, it isn't exactly misconfigured (like the issue subject says), you just want it to test with 3-way replication for the offsets topic, yes?